Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -422,6 +422,11 @@ public static Furniture PlaceInstance(Furniture proto, Tile tile)
         return obj;
     }
 
+    public bool WorkspotIsInternal()
```

Comment:
I don't think so, furniture is, to the best of my knowledge, the only thing that has workspots. Isn't BuildableJobs the jobs for building furnitures?